edgar_tool/text_search.py

```diff
@@ -197,7 +194,7 @@ def _generate_request_args(
 
         :param keywords: Search keywords to input in the "Document word or phrase" field
         :param entity_id: Entity/Person name, ticker, or CIK number to input in the "Company name, ticker, or CIK" field
-        :param filing_type: Filing category to select from the dropdown menu, defaults to None
+        :param filing_form: Group to select within the filing category dropdown menu, defaults to None
```

Comment:
This was a pre-existing documentation fault, but I think referring to the "dropdown menu" won't be clear to a user of the tool on the command line (as there is no dropdown).

Reply:
I can change it to:
`:param filing_form: Group to select within the filing category, defaults to None`

Reply:
Oh wait, I'm being silly. This relates to the SEC's EDGAR front-end, not our tool, it's replicating the inputs. No need to do anything here!